### ISSUE TITLE
Rule no-misspelled-properties not working for -webkit-touch-callout

### ISSUE THREAD

**Issue by bastienmoulia** [sasstools/sass-lint#1024]
When I use the properties `-webkit-touch-callout` i got a warning.

``` css
.my-class {
  -webkit-touch-callout: none;
}
```
```
warning  Property `touch-callout` appears to be spelled incorrectly                 no-misspelled-properties
```

It seems that the rule [no-misspelled-properties](https://github.com/sasstools/sass-lint/blob/b7bcb3a4ec1a9c08d3883eb8c2acb97b6e6513c3/lib/rules/no-misspelled-properties.js#L34) remove the `-webkit-` prefix and there is no `touch-callout` in the [properties.yml](https://github.com/sasstools/sass-lint/blob/b7bcb3a4ec1a9c08d3883eb8c2acb97b6e6513c3/data/properties.yml).

A quick fix is to add `touch-callout` in the `extra-properties` option in the .sass-lint.yml file:
``` yml
 no-misspelled-properties:
    - 1
    - extra-properties: ["touch-callout"]
```

But for long term, the rule should check for `-webkit-touch-callout` and this should be in the properties.yml list because `touch-callout` is not [standard](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-touch-callout).

Using:
- Sass-lint 1.10.2
- Visual Studio Code 1.9.1
- VSCode plugin Sass Lint 0.0.3
